docs/host-and-deploy/index.md

```diff
@@ -0,0 +1,162 @@
+  - *\*.wasm*: `application/wasm`
+  - *\*.woff*: `application/font-woff`
+  - *\*.woff2*: `application/font-woff`
+* HTTP compression is enabled for the following MIME types:
+  - `application/octet-stream`
+  - `application/wasm`
+* URL Rewrite Module rules are established:
+  - Serve the sub-directory where the app's static assets reside (`<assembly_name>\dist\<path_requested>`).
+  - Create SPA fallback routing so that requests for non-file assets are redirected to the app's default document in its static assets folder (`<assembly_name>\dist\index.html`).
+
+**Install the URL Rewrite Module**
+
+The [URL Rewrite Module](https://www.iis.net/downloads/microsoft/url-rewrite) is required to rewrite URLs. The module isn't installed by default, and it isn't available for install as an Web Server (IIS) role service feature. The module must be downloaded from the IIS website. Use the Web Platform Installer to install the module:
+
+1. Locally, navigate to the [URL Rewrite Module downloads page](https://www.iis.net/downloads/microsoft/url-rewrite#additionalDownloads). For the English version, select **WebPI** to download the WebPI installer. For other languages, select the appropriate architecture for the server (x86/x64) to download the installer.
+1. Copy the installer to the server. Run the installer. Select the **Install** button and accept the license terms. A server restart isn't required after the install completes.
+
+**Configure the website**
+
+Set the website's **Physical path** to the Blazor app's folder. The folder contains:
+
+* The *web.config* file that IIS uses to configure the website, including the required redirect rules and file content types.
+* The app's static asset folder.
+
+**Troubleshooting**
+
+If a *500 Internal Server Error* is received and IIS Manager throws errors when attempting to access the website's configuration, confirm that the URL Rewrite Module is installed. When the module isn't installed, the *web.config* file can't be parsed by IIS. This prevents the IIS Manager from loading the website's configuration and the website from serving Blazor's static files.
+
+For more information on troubleshooting deployments to IIS, see [Troubleshoot ASP.NET Core on IIS](https://docs.microsoft.com/aspnet/core/host-and-deploy/iis/troubleshoot).
+
+### Nginx
+
+The following *nginx.conf* file is simplified to show how to configure Nginx to send the *Index.html* file whenever it cannot find a corresponding file on disk.
+
+```
+events { }
+http {
+    server {
+        listen 80;
+
+        location / {
+            root /usr/share/nginx/html;
+            try_files $uri $uri/ /Index.html =404;
+        }
+    }
+}
+```
+
+For more information on production Nginx web server configuration, see [Creating NGINX Plus and NGINX Configuration Files](https://docs.nginx.com/nginx/admin-guide/basic-functionality/managing-configuration-files/).
+
+### Nginx in Docker
+
+Add one line to the Dockerfile, as shown in the following example:
```

Comment:
This says to modify a Dockerfile that I didn't even know I had yet. Can we add a sentence before this that provides some context? Maybe say something like: "To host Blazor in Docker using Nginx, setup your Dockerfile to use the Alpine based Nginx image. Then update the Dockerfile to copy your `nginx.config` file into the container.